lib/movie_card/movie_card.dart

```diff
@@ -13,75 +13,101 @@ class MovieCard extends StatelessWidget {
-        Navigator.of(context).push(
-          MaterialPageRoute(
-            builder: (context) => MovieDetailed(movie: movie),
-          ),
-        );
-      },
-      child: Container(
-        color: const Color.fromRGBO(23, 23, 23, 1),
-        height: 150,
-        child: Padding(
-          padding: const EdgeInsets.all(10),
-          child: Row(
-            children: [
-              Stack(
-                children: [
-                  Image.asset(movie.poster),
-                  const AddToWatchlistTab(),
-                ],
-              ),
-              Padding(
-                padding: const EdgeInsets.fromLTRB(10, 10, 0, 5),
-                child: Column(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  crossAxisAlignment: CrossAxisAlignment.start,
+        onTap: () {
+          Navigator.of(context).push(
+            MaterialPageRoute(
+              builder: (context) => MovieDetailed(movie: movie),
+            ),
+          );
+        },
+        child: Container(
+          color: const Color.fromRGBO(23, 23, 23, 1),
+          height: 150,
+          child: Padding(
+            padding: const EdgeInsets.all(10),
+            child: Row(
+              children: [
+                Stack(
                   children: [
-                    Padding(
-                      padding: const EdgeInsets.only(bottom: 5),
-                      child: Text(
-                        '${movie.topPosition}',
-                        style: const TextStyle(
-                          color: Colors.white,
-                          fontSize: 16,
+                    Image.asset(movie.poster),
+                    const AddToWatchlistTab(),
+                  ],
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(10, 10, 0, 5),
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(bottom: 5),
+                        child: Text(
+                          '${movie.topPosition}',
+                          style: const TextStyle(
+                            color: Colors.white,
+                            fontSize: 16,
+                          ),
                         ),
                       ),
-                    ),
-                    Padding(
-                      padding: const EdgeInsets.only(bottom: 5),
-                      child: Text(
-                        movie.title,
-                        style: const TextStyle(
-                          color: Colors.white,
-                          fontSize: 16,
+                      Padding(
+                        padding: const EdgeInsets.only(bottom: 5),
+                        child: Text(
+                          movie.title,
```

Comment:
Try to experiment with extremely long movie titles like: 'Extremely long movie title test yo yo yo etc etc etc asdfadfasdfasdfasdfasdfasdfasdf'.

How the Text will behave?